pennylane/plugins/default_qubit.py

```diff
@@ -296,7 +296,9 @@ class DefaultQubit(Device):
     Args:
         wires (int): the number of modes to initialize the device in
         shots (int): How many times the circuit should be evaluated (or sampled) to estimate
-            the expectation values. A value of 0 yields the exact result.
+            the expectation values. Defaults to 1 if not specified.
```

Comment:
There is a bit of a subtlety here.

* If `analytic=True`, ``shots`` is not used to calculate expval and var, but it does determine the number of samples returned.

* If `analytic = False`, ``shots`` _is_ used to calculate expval and var, and continues to determine the number of samples returned.

I think this could be made more clear in the docstring here (and in other relevant places)